api/routes/slot.py

```diff
@@ -28,10 +30,63 @@ def review_slot(user_id):
         update(Slot)
         .where(Slot.user_id == user_id)
         .where(Slot.slot_id == slot_id)
-        .where(Slot.end <= instant())
         .values(feedback=feedback)
     )
     db.session.execute(new_dur)
+
+    # if the session went badly or well, adjust the break period
+    # add on 5 minutes if reacted poorly, take off 5 mins if good
+    # adjust session after as well
+    if feedback == 1 or feedback == 3:
+        # get this slots end time
+        this_slot_q = (
+            select(Slot).where(Slot.user_id == user_id).where(Slot.slot_id == slot_id)
+        )
+        this_slot = db.session.execute(this_slot_q).scalars().first()
```

Comment:
one_or_none()

Reply:
What's the difference between that vs. `scalars().first()`. I can't see any usage of `one_or_none()` in the current codebase